aspnet/mvc/views/forms/sample/final/ViewModels/RegisterViewModelPW.cs

```diff
@@ -0,0 +1,17 @@
+﻿// define PW
+#if PW
+
+using System.ComponentModel.DataAnnotations;
+
+namespace FormsTH.ViewModels
+{
+    public class RegisterViewModel
```

Comment:
Why is a class named `RegisterViewModel` in a file named `RegisterViewModelPW` and what does `PW` stand for?